blocks/footer/footer.js

```diff
@@ -14,12 +14,35 @@ export default async function decorate(block) {
 
   if (resp.ok) {
     const html = await resp.text();
-
     // decorate footer DOM
     const footer = document.createElement('div');
-    footer.innerHTML = html;
-
+    footer.className = 'footer-container';
```

Comment:
are all these needed, or can you repurpose the divs/dom from the fetched footer html?

Reply:
I removed the footer-container div, so its child div footer-container-flex is the new parent wrapper div. I also moved footer-container's CSS to footer-container-flex.
Before:
```
<footer>
     <div class="footer block">
          <div class="footer-container">
              <div class="footer-container-flex">
                  ....
```
After:
```
<footer>
     <div class="footer block">
              <div class="footer-container-flex">
                  ....
```